src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,23 @@
 package core.basesyntax;
 
 public class MainApp {
+    public static void main(String[] args) {
+        Truck truck = new Truck();
 
+        Bulldozer bulldozer = new Bulldozer();
+
+        Excavator excavator = new Excavator();
```

Comment:
```suggestion
        Truck truck = new Truck();
        Bulldozer bulldozer = new Bulldozer();
        Excavator excavator = new Excavator();
```
зазвичай так код не розтягується, тільки якщо є логічно відділені участки в одному методі